changelog/4756.improvement.rst

```diff
@@ -0,0 +1,2 @@
+Make Rasa server response timeout configurable.
+Rasa server can now be started with `--response-timeout <int>` to configure a response timeout of <int> seconds.
```

Comment:
```suggestion
Rasa server can now be started with ``--response-timeout <int>`` to configure a 
response timeout of ``<int>`` seconds.
```

Reply:
It would be great if we could list all CLI commands that now support this new parameter

Reply:
> It would be great if we could list all CLI commands that now support this new parameter

I'm not sure how to do that.

Reply:
they're the same ones you already updated the test files for - `rasa x`, `rasa run` and `rasa shell` 🙂